x/genutil/client/cli/gentx.go

```diff
@@ -120,8 +120,8 @@ func GenTxCmd(ctx *server.Context, cdc *codec.Codec, mbm module.BasicManager, sm
 				return errors.Wrap(err, "failed to validate account in genesis")
 			}
 
-			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(authclient.GetTxEncoder(cdc))
-			clientCtx := client.NewContextWithInput(inBuf).WithCodec(cdc).WithJSONMarshaler(cdc)
```

Comment:
was there a reason `WithJSONMarshaler(cdc)` was removed?

Reply:
it seems to have been dropped when one of the merges with master was applied, probably when the integration tests started failing. I think what occurred is you pushed to your remote, which passed tests on github, then merged master and tested locally which started failing.

Reply:
thanks @colin-axner for spotting the error during the merge! Still after making your suggested changes, the integration test locally for me don't pass even though in gihhub is fine. I get a lof of this error message:
![Selection_068](https://user-images.githubusercontent.com/30392990/84818840-714f9a00-b017-11ea-8875-5384046c08d1.png)

Reply:
happy to help!

ah this is an issue with your local configuration. Looks like the tests are being denied access to port "32803", unfortunately I'm not too familiar with any reasons for why this would be. Could be worthwhile to hop on one of the community chats and ask for help if you want to try to fix it. However, it looks like the tests failing are for cli stuff so not a big deal if you can't run those locally. I'd just stick with `go test` for testing modified packages until you get around to fixing it